ceph.md

```diff
@@ -244,6 +250,40 @@ The `$FSID` value above should contain the actual FSID as described
 in the "Create a Ceph Secret" section. The FSID itself does not need
 to be considered secret.
 
+## Configure Manila with native CephFS
+
+Use a customServiceConfig to pass overrides to Manila's configuration
+file. For example, the sample
+[core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml)
+has the following in the OpenStackControlPlane CR:
```

Comment:
This will be a true statement after the following merges.

https://github.com/openstack-k8s-operators/openstack-operator/pull/332